pkg/util/status/build_status.go

```diff
@@ -71,10 +71,18 @@ const (
 
 	// ReasonArtifactsFetchFailed is the reason associated with a failure to
 	// download specified scripts in the application image.
-	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchScriptsFailed"
+	ReasonArtifactsFetchFailed api.StepFailureReason = "FetchArtifactsFailed"
 	// ReasonMessageArtifactsFetchFailed is the message associated with a failure to
 	// download specified scripts in the application image.
-	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch scripts specified scripts."
+	ReasonMessageArtifactsFetchFailed api.StepFailureMessage = "Failed to fetch specified scripts."
```

Comment:
Seems like this should be renamed to ReasonMessageScriptsFetchFailed (and ReasonScriptsFetchFailed above)